### PR TITLE
#4 python testing

### LINKED ISSUE #4: Construct ROS testing environment
**Motivation**

- Run test code before merging branches

**What should be realised**

```python
python3 -m pytest tests
```

### PR DESCRIPTION
**Pull request by r-yamada1998**
**Checks**

- [ ] Debugging completed
- [x] Please debug me

**Related Issue**

This PR closes #4.

### CODE REVIEW

File: necst/__init__.py [hunk 1]
```diff
@@ -8,4 +8,6 @@
 except:  # noqa: E722
     __version__ = "0.0.0"
 
-config = neclib.config
+#config = neclib.config
```
Comment (KaoruNishikawa) [inline necst/__init__.py:11]: Re-enable this, please.

**Review by KaoruNishikawa — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Please fix following points and where I've commented.

- There's an empty file `necst/tempos/ros2_string_sub.py`. Is this required to exist?

File: necst/__init__.py (hunk 1 above)
Comment (r-yamada1998) [reply necst/__init__.py]: Sorry, i've forgot to remove 'em

**Review by KaoruNishikawa — APPROVED** [review]
LGTM!

**Pull request merged**